sunpy/net/hek/hek.py

```diff
@@ -22,7 +25,9 @@
 __all__ = ['HEKClient', 'HEKTable', 'HEKRow']
 
 DEFAULT_URL = 'https://www.lmsal.com/hek/her?'
+HEK_FILE_PATH = 'sunpy/net/hek/hek_properties.json'
```

Comment:
This file path will need to be worked out at import and not hardcoded like this.

Reply:
@nabobalis could you give me an example on how you want this to be done. not really able to understand what to do here. thanks

Reply:
Similar to how https://github.com/sunpy/sunpy/blob/main/sunpy/data/test/__init__.py#L28 works

Reply:
added this change. please have a look

Reply:
Looks like that would work. I left a comment about the final path.